### PR TITLE
fix doctests broken by 36134

### PR DESCRIPTION
**Pull request by simeonschaub**
Looks like these were missed in #36134 
cc: @KristofferC

### CODE REVIEW

**Comment by KristofferC** [comment]
The doctests ran and passed there. Perhaps something was merged in between.

**Comment by simeonschaub** [comment]
Yes, that was my guess too.

File: doc/src/manual/constructors.md [hunk 1]
```diff
@@ -552,7 +552,8 @@ ERROR: MethodError: no method matching SummedArray(::Vector{Int32}, ::Int32)
 Closest candidates are:
   SummedArray(::Vector{T}) where T at none:4
 Stacktrace:
- [1] top-level scope at none:1
```
Comment (KristofferC) [inline doc/src/manual/constructors.md:555]: Can we just change these to `[...]`.

Comment (simeonschaub) [reply doc/src/manual/constructors.md:555]: Should the
```
 Closest candidates are:
  SummedArray(::Vector{T}) where T at none:4 
```
still be included? But maybe also not that important.

Comment (KristofferC) [reply doc/src/manual/constructors.md:555]: I think that can be kept.

**Comment by KristofferC** [comment]
Merging early so CI don't fail in other places.

**Pull request merged**